pkg/processor/build/runtime/java/runtime.go

```diff
@@ -0,0 +1,364 @@
+/*
+Copyright 2017 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package java
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/nuclio/nuclio/pkg/common"
+	"github.com/nuclio/nuclio/pkg/dockerclient"
+	"github.com/nuclio/nuclio/pkg/errors"
+	"github.com/nuclio/nuclio/pkg/processor/build/runtime"
+	"github.com/nuclio/nuclio/pkg/processor/build/util"
+	"github.com/nuclio/nuclio/pkg/version"
+
+	"github.com/rs/xid"
+)
+
+const (
+	buildFileName = "build.gradle"
+	// Must be in sync with pkg/processor/runtime/java/build.gradle and build dokcers
+	userHandlerJarName = "user-handler.jar"
+)
+
+var buildTemplateCode = `
```

Comment:
i think it's nicer to have this in its own source file - less clutter

Reply:
What do you mean by `this`? The template?